coordinator/src/context.rs

```diff
@@ -1,3 +1,19 @@
+// Copyright 2020 Kodebox, Inc.
```

Comment:
I think it's better to create a new PR for this commit.

Reply:
Okay. How about the commits after this? Should they be moved to a separate PR as well?

Reply:
I'll move all commits after the license commit to a separate branch.

Reply:
No. I meant only the license commit.

Reply:
No problem.